### ISSUE TITLE
Rebranding | Unreadable "Skip to content" link

### ISSUE THREAD

**Issue by tzi** [piwik/piwik#24536]
### What happened?

When we navigate with the keyboard, the link "skip to content" is not visible since the header redesign.

### What should happen?

The link "skip to content" should be visible

| *current* | *expected* |
|---|---|
| <img width="672" height="239" alt="Image" src="https://github.com/user-attachments/assets/eb6ff200-d58d-4eaa-b204-efae9aade533" /> | <img width="672" height="239" alt="Image" src="https://github.com/user-attachments/assets/4d7a9f99-7e10-42ce-95f8-0bce4c2b0eb8" /> |

### How can this be reproduced?

Load a Dashboard, do not click on the page with your mouse and press Tab.

### Matomo version

5.x

### Relevant log output

```shell

```

### Validations

- [x] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [x] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [x] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.
